aci/data_source_aci_cloudtemplatevpnnetwork.go

```diff
@@ -0,0 +1,153 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciCloudTemplateforVPNNetwork() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciCloudTemplateforVPNNetworkRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"aci_cloud_external_network_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"remote_site_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"remote_site_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"ipsec_tunnel": {
```

Comment:
Can you check if the sub-attributes below are actually Required?

Reply:
It is verifed.